metadata-ingestion/tests/integration/dbt/dbt_test_events_golden.json

```diff
@@ -1193,11 +1222,11 @@
 },
 {
     "entityType": "dataset",
-    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:postgres,calm-pagoda-323403.jaffle_shop.stg_customers,PROD)",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:postgres,calm-pagoda-323403.jaffle_shop.customers,PROD)",
```

Comment:
Did all of these have their name changed?

Reply:
nope- just a reordering of the json

only diff is that we're emitting status aspects for assertions now